config/rollup.config.js

```diff
@@ -36,7 +37,15 @@ const buildConfig = file =>
                         module: true,
                     }),
             sass({
-                     output: true,
+                output: function (styles, styleNodes) {
```

Comment:
@vforge @JoshuaRogan 

this is a breaking change. 

until now we're bundling all CSS files found in the tree of specific component, which resulted in duplicated content when eg. you've imported `Spinner` and `Select` (which also contains `Spinner` inside) => styles for Spinner were imported twice.

in Global Nav it was even more problematic:
firstly - we have 5+ components included inside, so bundled file included duplicated styles for Dropdowns, Avatars, Lists etc.
secondly - these CSS files were imported in random (well, maybe not really random, but clearly uncontrollable) way, resulting in sth like:
```
// Nav.css
...nav styles
...button styles
...dropdown styles
...eg. avatar styles
```
which was breaking current nav selectors because of their specificity.

With that being said - now we're bundling ONLY styles for "base" component (imported usually via `styles.scss`), so every client will be required to import ALL specific styles for every (sub)component one by one (if they're not imported yet) 

I've tried to dedupe it in different ways, including different rollup plugins or using `node-sass` directly, but this approach was imo the best/simplest, so lemme know if you have other ideas.

Reply:
Sounds good for now this will be solved once we move towards styled-components anyway.